syntax = "proto3";

package amarlon;

message SkillData {
	int32  id          = 1;
	int32  level       = 2;
	bool passive       = 3;
  int32 range        = 4;
  int32 radius       = 5;
	int32 target_type  = 6;
	string name        = 7;
	string description = 8;
}

message SkillsData {
	repeated SkillData skill = 1;
}